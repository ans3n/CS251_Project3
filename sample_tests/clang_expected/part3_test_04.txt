size
0
bucket_count
14
empty
true
insert Rayna Nerano Ondres Droth
size
1
insert Idula Ulvel Devas Omani
print
  0: [empty]
  1: [empty]
  2: [empty]
  3: [empty]
  4: [empty]
  5: [empty]
  6: [empty]
  7: [empty]
  8: Idula Ulvel -> Devas Omani
     └L: Rayna Nerano -> Ondres Droth
  9: [empty]
 10: [empty]
 11: [empty]
 12: [empty]
 13: [empty]
peek Rayna Nerano
Ondres Droth
bucket_count
14
insert Dralasa Dralayn Niras Selvayn
bucket_count
14
insert Goras Terandas Llaalamu Othren
peek Dralasa Dralayn
Niras Selvayn
insert Gindrala Tistar Galar Garil
size
5
insert Bedenea Sathren Guls Madryon
print
  0: [empty]
  1: Bedenea Sathren -> Guls Madryon
     └R: Goras Terandas -> Llaalamu Othren
  2: [empty]
  3: [empty]
  4: [empty]
  5: Dralasa Dralayn -> Niras Selvayn
  6: [empty]
  7: [empty]
  8: Rayna Nerano -> Ondres Droth
     └R: Idula Ulvel -> Devas Omani
  9: [empty]
 10: [empty]
 11: Gindrala Tistar -> Galar Garil
 12: [empty]
 13: [empty]
peek Dralasa Dralayn
Niras Selvayn
extract Rayna Nerano
Ondres Droth
peek Bedenea Sathren
Guls Madryon
empty
false
insert Ilen Andothan Dram Alam
insert Nervana Nelas Ralam Guls
hash_code Dralasa Dralayn
5
insert Melur Othreloth Gilas Gimayn
insert Farena Omaren Maralie Llothas
extract Dralasa Dralayn
Niras Selvayn
insert Garvs Orethi Sortis Sadus
print
  0: [empty]
  1: Ilen Andothan -> Dram Alam
     └R: Bedenea Sathren -> Guls Madryon
      └R: Goras Terandas -> Llaalamu Othren
  2: [empty]
  3: [empty]
  4: [empty]
  5: [empty]
  6: [empty]
  7: Nervana Nelas -> Ralam Guls
  8: Melur Othreloth -> Gilas Gimayn
     └R: Idula Ulvel -> Devas Omani
  9: Garvs Orethi -> Sortis Sadus
     └L: Farena Omaren -> Maralie Llothas
 10: [empty]
 11: Gindrala Tistar -> Galar Garil
 12: [empty]
 13: [empty]
size
9
empty
false
size
9
print
  0: [empty]
  1: Ilen Andothan -> Dram Alam
     └R: Bedenea Sathren -> Guls Madryon
      └R: Goras Terandas -> Llaalamu Othren
  2: [empty]
  3: [empty]
  4: [empty]
  5: [empty]
  6: [empty]
  7: Nervana Nelas -> Ralam Guls
  8: Melur Othreloth -> Gilas Gimayn
     └R: Idula Ulvel -> Devas Omani
  9: Garvs Orethi -> Sortis Sadus
     └L: Farena Omaren -> Maralie Llothas
 10: [empty]
 11: Gindrala Tistar -> Galar Garil
 12: [empty]
 13: [empty]
insert Tunila Llaram Alurue Ramarys
peek Bedenea Sathren
Guls Madryon
empty
false
print
  0: [empty]
  1: Bedenea Sathren -> Guls Madryon
     ├L: Ilen Andothan -> Dram Alam
     └R: Goras Terandas -> Llaalamu Othren
  2: [empty]
  3: [empty]
  4: [empty]
  5: [empty]
  6: [empty]
  7: Nervana Nelas -> Ralam Guls
  8: Melur Othreloth -> Gilas Gimayn
     └R: Idula Ulvel -> Devas Omani
  9: Tunila Llaram -> Alurue Ramarys
     └R: Farena Omaren -> Maralie Llothas
      └R: Garvs Orethi -> Sortis Sadus
 10: [empty]
 11: Gindrala Tistar -> Galar Garil
 12: [empty]
 13: [empty]
bucket_count
14
extract Ilen Andothan
Dram Alam
peek Bedenea Sathren
Guls Madryon
bucket_count
14
peek Bedenea Sathren
Guls Madryon
print
  0: [empty]
  1: Bedenea Sathren -> Guls Madryon
     └R: Goras Terandas -> Llaalamu Othren
  2: [empty]
  3: [empty]
  4: [empty]
  5: [empty]
  6: [empty]
  7: Nervana Nelas -> Ralam Guls
  8: Melur Othreloth -> Gilas Gimayn
     └R: Idula Ulvel -> Devas Omani
  9: Tunila Llaram -> Alurue Ramarys
     └R: Farena Omaren -> Maralie Llothas
      └R: Garvs Orethi -> Sortis Sadus
 10: [empty]
 11: Gindrala Tistar -> Galar Garil
 12: [empty]
 13: [empty]
insert Selman Reloro Nerile Sarothren
hash_code Farena Omaren
9
quit
