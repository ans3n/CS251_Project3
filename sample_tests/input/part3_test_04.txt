name name
14
size
bucket_count
empty
insert Rayna Nerano Ondres Droth
size
insert Idula Ulvel Devas Omani
print
peek Rayna Nerano
bucket_count
insert Dralasa Dralayn Niras Selvayn
bucket_count
insert Goras Terandas Llaalamu Othren
peek Dralasa Dralayn
insert Gindrala Tistar Galar Garil
size
insert Bedenea Sathren Guls Madryon
print
peek Dralasa Dralayn
extract Rayna Nerano
peek Bedenea Sathren
empty
insert Ilen Andothan Dram Alam
insert Nervana Nelas Ralam Guls
hash_code Dralasa Dralayn
insert Melur Othreloth Gilas Gimayn
insert Farena Omaren Maralie Llothas
extract Dralasa Dralayn
insert Garvs Orethi Sortis Sadus
print
size
empty
size
print
insert Tunila Llaram Alurue Ramarys
peek Bedenea Sathren
empty
print
bucket_count
extract Ilen Andothan
peek Bedenea Sathren
bucket_count
peek Bedenea Sathren
print
insert Selman Reloro Nerile Sarothren
hash_code Farena Omaren
quit
