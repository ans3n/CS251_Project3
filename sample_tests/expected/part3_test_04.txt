size
0
bucket_count
14
empty
true
insert Rayna Nerano Ondres Droth
size
1
insert Idula Ulvel Devas Omani
print
  0: [empty]
  1: [empty]
  2: [empty]
  3: Idula Ulvel -> Devas Omani
  4: [empty]
  5: [empty]
  6: Rayna Nerano -> Ondres Droth
  7: [empty]
  8: [empty]
  9: [empty]
 10: [empty]
 11: [empty]
 12: [empty]
 13: [empty]
peek Rayna Nerano
Ondres Droth
bucket_count
14
insert Dralasa Dralayn Niras Selvayn
bucket_count
14
insert Goras Terandas Llaalamu Othren
peek Dralasa Dralayn
Niras Selvayn
insert Gindrala Tistar Galar Garil
size
5
insert Bedenea Sathren Guls Madryon
print
  0: [empty]
  1: [empty]
  2: [empty]
  3: Bedenea Sathren -> Guls Madryon
     └R: Goras Terandas -> Llaalamu Othren
      └R: Idula Ulvel -> Devas Omani
  4: [empty]
  5: [empty]
  6: Rayna Nerano -> Ondres Droth
  7: [empty]
  8: [empty]
  9: Gindrala Tistar -> Galar Garil
 10: [empty]
 11: Dralasa Dralayn -> Niras Selvayn
 12: [empty]
 13: [empty]
peek Dralasa Dralayn
Niras Selvayn
extract Rayna Nerano
Ondres Droth
peek Bedenea Sathren
Guls Madryon
empty
false
insert Ilen Andothan Dram Alam
insert Nervana Nelas Ralam Guls
hash_code Dralasa Dralayn
11
insert Melur Othreloth Gilas Gimayn
insert Farena Omaren Maralie Llothas
extract Dralasa Dralayn
Niras Selvayn
insert Garvs Orethi Sortis Sadus
print
  0: [empty]
  1: Ilen Andothan -> Dram Alam
  2: [empty]
  3: Bedenea Sathren -> Guls Madryon
     └R: Goras Terandas -> Llaalamu Othren
      └R: Idula Ulvel -> Devas Omani
  4: Farena Omaren -> Maralie Llothas
  5: [empty]
  6: [empty]
  7: [empty]
  8: [empty]
  9: Gindrala Tistar -> Galar Garil
 10: Nervana Nelas -> Ralam Guls
 11: Melur Othreloth -> Gilas Gimayn
 12: [empty]
 13: Garvs Orethi -> Sortis Sadus
size
9
empty
false
size
9
print
  0: [empty]
  1: Ilen Andothan -> Dram Alam
  2: [empty]
  3: Bedenea Sathren -> Guls Madryon
     └R: Goras Terandas -> Llaalamu Othren
      └R: Idula Ulvel -> Devas Omani
  4: Farena Omaren -> Maralie Llothas
  5: [empty]
  6: [empty]
  7: [empty]
  8: [empty]
  9: Gindrala Tistar -> Galar Garil
 10: Nervana Nelas -> Ralam Guls
 11: Melur Othreloth -> Gilas Gimayn
 12: [empty]
 13: Garvs Orethi -> Sortis Sadus
insert Tunila Llaram Alurue Ramarys
peek Bedenea Sathren
Guls Madryon
empty
false
print
  0: Tunila Llaram -> Alurue Ramarys
  1: Ilen Andothan -> Dram Alam
  2: [empty]
  3: Bedenea Sathren -> Guls Madryon
     └R: Goras Terandas -> Llaalamu Othren
      └R: Idula Ulvel -> Devas Omani
  4: Farena Omaren -> Maralie Llothas
  5: [empty]
  6: [empty]
  7: [empty]
  8: [empty]
  9: Gindrala Tistar -> Galar Garil
 10: Nervana Nelas -> Ralam Guls
 11: Melur Othreloth -> Gilas Gimayn
 12: [empty]
 13: Garvs Orethi -> Sortis Sadus
bucket_count
14
extract Ilen Andothan
Dram Alam
peek Bedenea Sathren
Guls Madryon
bucket_count
14
peek Bedenea Sathren
Guls Madryon
print
  0: Tunila Llaram -> Alurue Ramarys
  1: [empty]
  2: [empty]
  3: Bedenea Sathren -> Guls Madryon
     └R: Goras Terandas -> Llaalamu Othren
      └R: Idula Ulvel -> Devas Omani
  4: Farena Omaren -> Maralie Llothas
  5: [empty]
  6: [empty]
  7: [empty]
  8: [empty]
  9: Gindrala Tistar -> Galar Garil
 10: Nervana Nelas -> Ralam Guls
 11: Melur Othreloth -> Gilas Gimayn
 12: [empty]
 13: Garvs Orethi -> Sortis Sadus
insert Selman Reloro Nerile Sarothren
hash_code Farena Omaren
4
quit
